rpc/src/rpc.rs

```diff
@@ -154,6 +157,28 @@ pub struct JsonRpcConfig {
     pub disable_health_check: bool,
 }
 
+impl Default for JsonRpcConfig {
+    fn default() -> Self {
+        Self {
+            enable_rpc_transaction_history: Default::default(),
+            enable_extended_tx_metadata_storage: Default::default(),
+            faucet_addr: Option::default(),
+            health_check_slot_distance: Default::default(),
+            rpc_bigtable_config: Option::default(),
+            max_multiple_accounts: Option::default(),
+            account_indexes: AccountSecondaryIndexes::default(),
+            rpc_threads: 1,
```

Comment:
Should I add a dep on num_cpus here so I can num_cpus::get()?

Reply:
could. i doubt that's what we want long term*, but no dumber that what we do today.

\* i'm looking into making all our thread pools cli-configurable atm :hidethepainharold:

Reply:
then I'll leave as is - the validator always overrides these fields anyway, the default values are only used in tests